### DISCUSSION TITLE
Spatial and Temporal Inpute scale

### DISCUSSION THREAD

**Discussion by dang-trinh156** [lululxvi/deepxde#1300] [Q&A] [answered]
Dear DeepXDE developers and users,
As far as I know, almost all the examples of DeepXDE, and vast majority of PINN papers use spatial domain either [0,1] or [-1,1] for 1D or 3D, and time domain [0,1]. 
In physical domains, it should be something like [1000 ft,10000 ft] or [30m,3000m] or [10 days, 2000 days] or something far bigger than [0,1]. 
If that's the normalization of spatial domain, wouldn't it make incorrections in PDEs' units, and thus, lead to be unable to learn the physics?
Thank you for your answers!!
p/s: I've gone through all the cases in FAQ Q: I failed to train the network or get the right solution, e.g., large training loss, unbalanced losses. But I've found no clear answer about this.

**Comment by lululxvi** [comment, marked as answer]
https://en.wikipedia.org/wiki/Nondimensionalization

**Reply by dang-trinh156** [reply]
Oh, thank you! That's what I've been wondering
